configure.ac

```diff
@@ -191,6 +197,24 @@ AC_DEFINE([NETDATA_WITH_UUID], [1], [uuid settings])
 OPTIONAL_UUID_CLFAGS="${UUID_CFLAGS}"
 OPTIONAL_UUID_LIBS="${UUID_LIBS}"
 
+if test ! "${enable_plugin_freeipmi}" = "no"; then
+	PKG_CHECK_MODULES(
+		[IPMIMONITORING],
+		[libipmimonitoring],
+		,
+		:
+	)
+	if test ! -z "${IPMIMONITORING_LIBS}"; then
+		enable_plugin_freeipmi="yes"
+		AC_DEFINE([HAVE_FREEIPMI], [1], [ipmimonitoring usability])
+		OPTIONAL_IPMIMONITORING_CLFAGS="${IPMIMONITORING_CFLAGS}"
+		OPTIONAL_IPMIMONITORING_LIBS="${IPMIMONITORING_LIBS}"
+	elif test "${enable_plugin_freeipmi}" = "yes"; then
+		AC_MSG_ERROR([ipmimonitoring required but not found. Try installing 'libipmimonitoring-dev' or 'libipmimonitoring-devel'])
+	fi
+fi
+AM_CONDITIONAL([FREEIPMI], [test x$enable_plugin_freeipmi = xyes])
```

Comment:
please rename FREEIPMI to ENABLE_PLUGIN_FREEIPMI so you can be consistent with other plugins such as ENABLE_PLUGIN_APPS